index.js

```diff
@@ -39,6 +39,11 @@ function filterResponseForImportantEvents(allEventsFromFetch) {
         const event = allEventsFromFetch[i];
         if (githubImportantEvents.indexOf(event.type) !== -1) {
             arrayOfImportantEvents.push(event);
+        } else if (event.payload) {
```

Comment:
As Kevin mentioned, this is a great implementation of the feature!

Reading through the code, it was hard for me to grasp the logic at first because we're doing comparisons of two similar things, one after the other. (Like, the second conditional doesn't actually rely on the first one, it just happens to come after it and I thought at first that it did rely on it.)

Personally, I think it might be clearer if you created `typeWithAction` and a new variable `type` on line 39.5 and then the conditional was ```if (githubImportantEvents.indexOf(type) !== -1 || githubImportantEvents.indexOf(typeWithAction) !== -1 )```
This has the added benefit of not repeating the push code.

What do you think? I'd also be willing to merge it as is or maybe you have another idea.

Reply:
Minor note: I prefer to use >= 0 to test the result of indexOf, rather than relying on it being exactly -1. Probably a holdover from my earlier days coding in C.